draft-regext-rdap-x-media-type.md

```diff
@@ -127,6 +136,117 @@ though clients SHOULD list the extension identifier in the "extensions" paramete
 other protocol elements of those extensions. Servers SHOULD NOT require the usage of extension
 identifiers in the "extensions" parameter when other extension protocol elements are used.
 
+## Examples
+
+The following examples use the HTTP/1.1 message exchange syntax as seen in [@!RFC9110].
+
+### Classic Negotiation
+
+This example demonstrates the negotiation of the "applicaton/rdap+json" media type
+as defined in [@!RFC7480] using an RDAP "/help" query. This example also demonstrates
+the negotiation in which a client does not support RDAP-X but a server does support
+RDAP-X.
+
+Client Request:
+
+    GET /help HTTP/1.1
+    accept: application/rdap+json
+
+Server Response:
+
+    HTTP/1.1 200 OK
+    content-type: application/rdap+json
+
+    { "rdapConformance" : [ "rdap_level_0" ],
+      "notices" : [
+        { "description" : [ "my content includes a trailing CRLF" ] } ] }
+
+
+### RDAP-X Negotiation
+
+In this example, both the client and server support RDAP-X and a fictional
+extension of "foo".
+
+Client Request:
+
+    GET /help HTTP/1.1
+    accept: application/rdap+json, application/rdap-x+json;extensions="rdap_level_0 rdapx foo"
+
+Server Response:
+
+    HTTP/1.1 200 OK
+    content-type: application/rdap-x+json;extensions="rdap_level_0 rdapx foo"
+
+    { "rdapConformance" : [ "rdap_level_0", "rdapx", "foo" ],
+      "notices" : [
+        { "description" : [ "my content includes a trailing CRLF" ] } ] }
+
+### No Server Support Negotiation
+
+In this example, only the client supports RDAP-X, along with a fictional
+extension of "foo".
+
+Client Request:
+
+    GET /help HTTP/1.1
+    accept: application/rdap+json, application/rdap-x+json;extensions="rdap_level_0 rdapx foo"
+
+Server Response:
+
+    HTTP/1.1 200 OK
+    content-type: application/rdap+json
+
+    { "rdapConformance" : [ "rdap_level_0", "rdapx", "foo" ],
+      "notices" : [
+        { "description" : [ "my content includes a trailing CRLF" ] } ] }
+
+### Differing Extension Negotiation
+
+In this example, both the client and server support RDAP-X. The client
+supports the extensions "foo" and "bar" while the server only support "foo".
+
+Client Request:
+
+    GET /help HTTP/1.1
+    accept: application/rdap+json, application/rdap-x+json;extensions="rdap_level_0 rdapx foo bar"
+
+Server Response:
+
+    HTTP/1.1 200 OK
+    content-type: application/rdap-x+json;extensions="rdap_level_0 rdapx foo"
+
+    { "rdapConformance" : [ "rdap_level_0", "rdapx", "foo" ],
+      "notices" : [
+        { "description" : [ "my content includes a trailing CRLF" ] } ] }
+
+### Extension Versioning and Meta-data {#versioning}
+
+For scenarios where the "versioning" extension, as defined by [@?I-D.ietf-regext-rdap-versioning],
+is used, the extension identifiers in the client request may not be exact or case-insensitive matches for the
```

Comment:
Confused about the "exact or case-insensitive matches". How does that factor in here?

Reply:
i think I addressed this.